dask_expr/_collection.py

```diff
@@ -145,6 +145,18 @@ def __getitem__(self, other):
             return new_collection(self.expr.__getitem__(other.expr))
         return new_collection(self.expr.__getitem__(other))
 
+    def persist(self, fuse=True, combine_similar=True, **kwargs):
+        out = self.optimize(combine_similar=combine_similar, fuse=fuse)
+        return DaskMethodsMixin.persist(
+            out, combine_similar=combine_similar, fuse=fuse, **kwargs
```

Comment:
Why are `combine_similar` and `fuse` passed into `DaskMethodsMixin.persist` like this? Same question for `compute` below.